tests/utils/LHNTestUtils.tsx

```diff
@@ -119,16 +127,26 @@ let lastFakeReportActionID = 0;
 /**
  * @param millisecondsInThePast the number of milliseconds in the past for the last message timestamp (to order reports by most recent messages)
  */
-function getFakeReport(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false): Report {
+function getFakeReport(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false, shouldAddParticipantRole: boolean = false): Report {
     const lastVisibleActionCreated = DateUtils.getDBTime(Date.now() - millisecondsInThePast);
 
+    const participants: Record<number, Participant> = {};
+
+    participantAccountIDs.forEach((id) => {
+        participants[id] = {
+            hidden: false,
+            role: shouldAddParticipantRole ? (id === 1 ? 'admin' : 'member') : null,
```

Comment:
This is still not valid for a util function. Not all participants are expected to have the admin using account id 1. Instead of `shouldAddParticipantRole` use a param that's an array and tells us the account ids of the admins.